### DISCUSSION TITLE
How cab i test repository class' functions

### DISCUSSION THREAD

**Discussion by emreakay** [pestphp/pest#435] [General]
hi,

i want to write unit test for my repository class functions like update, create, delete etc..

 I got this error

![image](https://user-images.githubusercontent.com/794216/140722299-f44d33c3-e173-4bda-b322-7798f03185ed.png)

how can i write it?

Thanks

**Comment by nunomaduro** [comment]
Seems that you are using the Laravel Framework in that test. Therefore, you need to boot the framework in your unit tests by adding `Unit` to your `Pest.php` file:

```php
/*
|--------------------------------------------------------------------------
| Test Case
|--------------------------------------------------------------------------
|
| The closure you provide to your test functions is always bound to a specific PHPUnit test
| case class. By default, that class is "PHPUnit\Framework\TestCase". Of course, you may
| need to change it using the "uses()" function to bind a different classes or traits.
|
*/

uses(Tests\TestCase::class)->in('Feature', 'Unit');
```

**Reply by emreakay** [reply]
you saved my day thanks :+1:
